src/index.js

```diff
@@ -56,7 +60,8 @@ class ReactTooltip extends Component {
     disable: PropTypes.bool,
     scrollHide: PropTypes.bool,
     resizeHide: PropTypes.bool,
-    wrapper: PropTypes.string
+    wrapper: PropTypes.string,
+    onPlaceChange: PropTypes.func // called when tooltip switches between any two places
```

Comment:
Can you update the `readme.md` to include this prop in its table? The column with the "Specific" heading would probably be blank.